service_management/azure/test/integration/test_helper.rb

```diff
@@ -35,7 +35,7 @@
   config.default_cassette_options = {:record => :once}
   config.cassette_library_dir = "vcr/vcr_cassettes"
   config.hook_into :faraday
-  config.allow_http_connections_when_no_cassette = false
+  config.allow_http_connections_when_no_cassette = true

```

Comment:
I believe that we need this line to make sure recorded tests never makes requests on network. We need this to be false. Thanks for catching this Fixing it on merge.